accounts/serializers.py

```diff
@@ -40,4 +40,39 @@ class UserInfoTransSerializer(serializers.Serializer):
     id = serializers.IntegerField()
 
     def get_full_name(self, obj):
-        return obj.first_name + " " + obj.last_name
\ No newline at end of file
+        return obj.first_name + " " + obj.last_name
+
+
+class UserSerializerAdminAccess(serializers.ModelSerializer):
+	password = serializers.CharField(write_only=True)
+	def create(self, validated_data):
+		is_active = validated_data.get('is_active')
+		is_staff = validated_data.get('is_staff')
+		is_superuser = validated_data.get('is_superuser')
+		if not is_active:

```

Comment:
Just set the default value when trying to get is_active. .get('is_active', False). Also is_active should be set to True by default so there should be no need to set it to true again.